src/drivers/iio_imu/driver.rs

```diff
@@ -1,8 +1,16 @@
-use std::{collections::HashMap, error::Error};
+use std::{
+    collections::{HashMap, HashSet},
+    error::Error,
+    fs::File,
+    io::{self, BufRead, BufReader},
+};
 
 use industrial_io::{Channel, ChannelType, Device, Direction};
 
-use crate::drivers::iio_imu::info::MountMatrix;
+use crate::{
+    drivers::iio_imu::info::MountMatrix,
+    input::capability::{Capability, Source},
```

Comment:
Ideally drivers should be standalone modules that do not rely on any InputPlumber data types. I would like to eventually move each of these drivers into their own crate that does not have any InputPlumber dependencies.